Gemfile

```diff
@@ -37,10 +27,7 @@ gem 'jbuilder', '~> 2.5'
 
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
-
-# You know, so things looks pretty
-gem 'bootstrap', '~> 4.3.1'
-gem 'jquery-rails'
+gem 'webpacker', '~> 4.0'
```

Comment:
a comment (like what you put in this PR) re: "why this gem?" wouldn't go amiss ...

Reply:
I've added the default comment that a newly generated rails application would have.  How does that sound? https://github.com/rails/rails/blob/0196551e6039ca864d1eee1e01819fcae12c1dc9/railties/lib/rails/generators/app_base.rb#L318

Reply:
perfect!